background/KF.ts

```diff
@@ -601,17 +602,17 @@ class KeeFox {
     RegularKPRPCListenerQueueHandler ()
     {
         // If there is nothing in the queue at the moment or we are already processing a callback, we give up for now
-        if (keefox_org.processingCallback || keefox_org.pendingCallback == "")
+        if (this.processingCallback || this.pendingCallback == "")
```

Comment:
keefox_org works for me; perhaps this does too but I'd prefer not to change working code in this way at the moment. Did you find some specific problem that caused you to change this code?

I have recently found that the way I was initialising configuration information was not correct and I've fixed that in the latest commits. I thought it would only result in always using the default configuration but it's possible the startup order was causing keefox_org to not be available at some early point of the startup process.